src/shared/App.js

```diff
@@ -31,83 +24,31 @@ function App() {
   }, []);
 
   return (
-    <ChakraProvider theme={dashboardTheme}>
-      {authenticating ? (
-        <Flex direction="column" alignItems="center">
-          <div>Authenticating... </div>
-          <Spinner size="xl" />
-        </Flex>
-      ) : (
-        <Router>
-          <Switch>
-            <Route exact path="/login">
-              {user ? <Redirect to="/" /> : <LoginRegister />}
-            </Route>
-            <Route exact path="/chakraExpoDashboard">
-              <ChakraExpoDashboard />
-            </Route>
-            <Route exact path="/chakraExpoStore">
-              <ChakraProvider theme={storeTheme}>
-                <ChakraExpoStore />
-              </ChakraProvider>
-            </Route>
-            <Route exact path="/store">
-              <Storefront />
-            </Route>
-            <ProtectedRoute path="/">
-              <Grid templateColumns={`${NAVBAR_WIDTH} 1fr`} h="100vh">
-                <Box borderRight="2px solid black" w="100%" h="100%">
-                  {/** Navbar width is set manually to keep the position fixed */}
-                  <Navbar w={NAVBAR_WIDTH} />
-                </Box>
-
-                <Box w="100%" h="100%" p="72px">
-                  <Switch>
-                    <ProtectedRoute exact path="/home">
-                      <Dashboard />
-                    </ProtectedRoute>
-                    <ProtectedRoute exact path="/leaderboard">
-                      Leaderboard page, not yet implemented.
-                    </ProtectedRoute>
-                    <ProtectedRoute exact path="/team">
-                      <TeamView />
-                    </ProtectedRoute>
-                    {/* All other paths are redirected to /home */}
-                    <ProtectedRoute path="/">
-                      <Redirect to="/home" />
-                    </ProtectedRoute>
-                  </Switch>
-                </Box>
-              </Grid>
-            </ProtectedRoute>
-          </Switch>
-        </Router>
-      )}
-    </ChakraProvider>
+    <BrowserRouter>
+      <Switch>
+        {/* pages with Chakra components, helpful to reference when developing */}
+        <Route exact path="/chakraExpoDashboard">
+          <ChakraProvider theme={dashboardTheme}>
+            <ChakraExpoDashboard />
+          </ChakraProvider>
+        </Route>
+        <Route exact path="/chakraExpoStore">
+          <ChakraProvider theme={storeTheme}>
+            <ChakraExpoStore />
+          </ChakraProvider>
+        </Route>
+
+        {/* TODO: put in storefront router */}
```

Comment:
@jzxhuang I can implement the storefront router in my ticket unless you had another ticket for this?

Reply:
@patrick-du yeah sure! I would keep it very sipmle/basic though (so just the routes you need) and have it as a starting point we can build on. I saw you removed `exact` already which is great!

There will be an entire ticket for more complex storefront routing (i.e. designing a URL structure with `teamId` in the URL). This will be a totally separate ticket on its own, since we probably want to have top-level logic that fetches the team based on the URL, puts it in Redux, etc. https://www.notion.so/uwblueprintexecs/Building-Up-be1b30c321554bffba37a64c6cdc1ec6?p=17c17e34752b4bdc8f787f803cde2998